plugins/partiql-plugin/src/main/kotlin/org/partiql/plugin/internal/fn/scalar/FnIsInt16.kt

```diff
@@ -21,6 +23,6 @@ internal object FnIsInt16 : PartiQLFunction.Scalar {
     )
 
     override fun invoke(args: Array<PartiQLValue>): PartiQLValue {
-        TODO("Function is_int16 not implemented")
+        return boolValue(args[0] is Int16Value)
```

Comment:
If lhs is a int8, should the function returns true?

i.e., `1 is INT16` should this be true?

More generally, I would think if the rhs is a super type of the value type, then the function should be true.

Reply:
Yes, considering range checks too